paasta_tools/kubernetes_tools.py

```diff
@@ -2283,6 +2288,8 @@ def create_kubernetes_secret_signature(
 
 
 def sanitise_kubernetes_name(service: str,) -> str:
+    if not service:
```

Comment:
Reason for adding this:
`paasta-elasticsearches` as service and instance as [null](https://fluffy.yelpcorp.com/i/wND4SG2TpflSqhh6jKgVN1qmngxkLshh.html
).

Reply:
Instead of allowing this error, I think we handle this case in the script:  https://github.com/Yelp/paasta/blob/master/paasta_tools/contrib/get_running_task_allocation.py#L139

Reply:
After moving the check to above line, test result: https://fluffy.yelpcorp.com/i/jnZTg0bXRr5l6dGmN3QvjB0hc5f6TJph.html#L60-62